src/ngraph_encapsulate_op.cc

```diff
@@ -213,6 +219,29 @@ class NGraphEncapsulateOp : public OpKernel {
     return Status::OK();
   }
 
+  void mem_usage(double& vm_usage, double& resident_set) {
```

Comment:
We can remove this function. It was moved to ngraph_utils.h and renamed to MemoryProfile. 

https://github.com/NervanaSystems/ngraph-tf/blob/master/src/ngraph_utils.cc#L247